src/fromager/commands/bootstrap.py

```diff
@@ -152,6 +158,62 @@ def bootstrap(
     logger.info(f"writing installation dependencies to {constraints_filename}")
     with open(wkctx.work_dir / "constraints.txt", "w") as f:
         write_constraints_file(graph=wkctx.dependency_graph, output=f)
+    _show_summary(wkctx)
+
+
+def _show_summary(wkctx: context.WorkContext) -> None:
```

Comment:
Not sure if we need this for bootstrap. The build order file essentially presents the same data but in a different format

Reply:
I'm using bootstrap for testing new builds and changes a lot. A summary table is very useful to understand the bootstrap result w/o parsing through JSON.

Reply:
There is a summary command, maybe we can shift this there?

Reply:
I have removed the summary from the bootstrap command. Will add it back in later PR.